config/credentials.yml.enc

```diff
@@ -1 +1 @@
-hD256W9DOruHc+WWJsemztuRbOTtIdwWMijt9EjrwhkJtT8u4gyKwdH4UmGBH7kWJsZ2kXiUTKd47U2yZl1JUjQHOGAd8J5SJ7JRLzRSt7vDiBU88aRyBCcH4nHnYgdfUw7dIYsJpRmSnh6GJeVl8/MXoQ27EGBmWxkUj1Xyf+Hj2c5Ousr527qHa2fg4mFDfeNUU/4RmNrcH1Tn0hYsBpLFbruYDsNi8oR4sd/hI11ZwIRuek9fM4ntG4b0AUFJ/yeZc/YeTv406TryaWuTKYBDBvx7kDt+PXn4cINBAZ/fnXA1CBUgGm1en99XZ0cvYvOf6y/pr+nCTFpUFK/fH0qZT+YEfL8BuuFFlj4HmCtfOy5NDvJyZRSHIHsgDtarU1hzDnsjjMP8//j6sv1GRMF/0HK5O/sOi3yd--Vh9tajLNHRkQ/bub--EGxqqDcBu4aRJOmWIXyCRw==
\ No newline at end of file
+9xHvCUpx0abPgWH9qrCObjba4XSoRPPWODmP0KcQcJmT0RvqNH1gdREjy+h9rW68QKXvEs25NXwsUEJPIIrBhucVzSieXd7/+YQrOj2jxP77WfqtksgTPdRyi5D3qCn05H6kFI1G/cQJ5S2PWXHpF+iRQZxiKucPxkqyr1FaPPgPOGT0j33ldPNn/wr+mdzqOAIiz9NhCoYgchnAVOmmuW5UrcrvC514QksswNEGu+5lT49BdF8aDFobqPe93mvoE50cR8vZ9cK3ZypTZPJFoBeeknSNKTPr3WPto758lvPEtS/OpKB1M8BYs3ed3H2+4X2y41LTYNj/ZKg+b6OaHkfaSZJR2gwmNMX5O+GfKCj3MBMN/NvnqPa8t17aAJGW+ZWkLjCTufdyGqcLsnx70pEi7aAAp0+p+YGL77phQ8Ln/bsSn1Qf1boEGoUSR1V4j5h7KWUn/tuNCX/xZQdnx1vEuUFgfQFdRzFTPcnKCviGA37cSlPkGKdBPL8W6nF4rJHaTSrc4AT8dbwP8zzkEhNYGp3Q5yMqyUZ/NgpUPcebGX6VgVyuyQQWYbPP3hwwZBeOHwNXu4K9ZM6t7ythtIbDCtTIwg==--VU3BybioZSUj8AjO--H/+Rp0R0GCwbaXEqmypUBw==
```

Comment:
I guess this shouldn't be here. We should store it as a secret

Reply:
It's not a secret. Instead, it contains some hidden properties that can be decrypted with a secret, which I shared with the whole team and is included in github as well